src/gui/toolbarMenubar/ColorToolItem.cpp

```diff
@@ -64,24 +64,13 @@ void ColorToolItem::enableColor(int color) {
             gtk_toggle_tool_button_set_active(GTK_TOGGLE_TOOL_BUTTON(this->item), false);
         }
     } else {
-        bool active = colorEqualsMoreOreLess(color);
-
         if (this->item) {
-            gtk_toggle_tool_button_set_active(GTK_TOGGLE_TOOL_BUTTON(this->item), active);
-        }
-
-        if (active) {
-            this->toolHandler->setColorFound();
-
-            // Only equals more ore less, so we will set it exact to the default color
-            if (this->color != color) {
-                this->toolHandler->setColor(this->color, true);
-            }
```

Comment:
the function ToolHandler::setColorFound() is not used anymore.
Remove it with the know obsolete colorFound variable.